docs/mutations.md

```diff
@@ -292,6 +386,9 @@ For instance, if we have
 
 ```{code-block} python
 
+import numpy as np
+from scipy import linalg
```

Comment:
I'd find it clearer to import scipy and call ``Pt = scipy.linalg.expm(t * rate * Q)`` below. Then you know "this function is from scipy" without needing to keep all the details of the imports in your head.